internal/core/domain/trade_service.go

```diff
@@ -73,12 +73,11 @@ func (t *Trade) Accept(
 	t.SwapAccept.Message = swapAcceptMsg
 	t.Timestamp.Accept = uint64(time.Now().Unix())
 	t.PsetBase64 = psetBase64
-
-	p, err := pset.NewPsetFromBase64(psetBase64)
+	txID, err := transactionutil.GetTxIdFromPset(psetBase64)
 	if err != nil {
-		return false, err
+		return false, nil
 	}
```

Comment:
we can skip checking for error because we know that the pset is wall formed here